src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/WebserviceController.php

```diff
@@ -66,17 +68,63 @@ public function indexAction(WebserviceKeyFilters $filters, Request $request)
 
         $configurationWarnings = $this->lookForWarnings();
 
+        $webserviceConfiguration = $this->get('prestashop.admin.webservice.form_data_provider')->getData();
+        $webserviceIsEnabled = $webserviceConfiguration['webservice_configuration']['enable_webservice'];
+        $webserviceStatus = [
+            'enabled' => $webserviceIsEnabled,
+            'isFunctional' => true,
+            'endpoint' => null,
+        ];
+
+        if ($webserviceIsEnabled) {
+            $expectedWebserviceEntryPoint = $request->getSchemeAndHttpHost() . self::WEBSERVICE_ENTRY_ENDPOINT;
+            $webServiceEntryPointIsReachable = $this->checkWebserviceEndpoint($expectedWebserviceEntryPoint);
+
+            $webserviceStatus['endpoint'] = $expectedWebserviceEntryPoint;
+            $webserviceStatus['isFunctional'] = $webServiceEntryPointIsReachable;
+        }
+
         return $this->render(
             '@PrestaShop/Admin/Configure/AdvancedParameters/Webservice/index.html.twig',
             [
                 'help_link' => $this->generateSidebarLink($request->get('_legacy_controller')),
                 'form' => $form->createView(),
                 'grid' => $presentedGrid,
                 'configurationWarnings' => $configurationWarnings,
+                'webserviceStatus' => $webserviceStatus,
             ]
         );
     }
 
+    /**
+     * @param string $url
+     *
+     * @return bool
+     */
+    private function checkWebserviceEndpoint($url): bool
```

Comment:
```suggestion
    private function checkWebserviceEndpoint(?string $url): bool
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/19335/commits/db6866eec2cf932f050ef74981bd353c4e967f52